### DISCUSSION TITLE
6714fc1dfa2f0dd3a91d2988

### DISCUSSION THREAD

**Discussion by giscus** [Admin-ToxicGames/toxicgames#2] [General]
# 6714fc1dfa2f0dd3a91d2988

Discover a wide range of cracked games, mods, and resources at Toxic Games. Download for free and dive into endless gaming adventures!

http://localhost:5173/6714fc1dfa2f0dd3a91d2988

**Comment by Admin-ToxicGames** [comment]
![Image of MacBook Air](https://img.playbook.com/5B0zR49dpXwQIsQ0bnI42WNJI2PGr91bbjtzrbqkQxM/Z3M6Ly9wbGF5Ym9v/ay1hc3NldHMtcHVi/bGljLzg4YTQxMWMz/LWI5YTYtNDBlZS1i/NTY3LTdmOTljYTIz/MTRhYg)

### Tested on a MacBook Air M1 with high settings, gives stable 40 FPS🔥.

**Reply by rootparthb** [reply]
its not verifeing on my mac

**Reply by Admin-ToxicGames** [reply]
It takes time, if it took more that 15 mins just close and restart.

**Reply by rootparthb** [reply]
its showing it has malware

**Reply by Admin-ToxicGames** [reply]
There is a code above the download button paste in the terminal, and allow application from anywhere from "privacy and security".

**Comment by lovelora123** [comment]
when will the dlc update come
?

**Reply by Admin-ToxicGames** [reply]
Didn't uploaded, maybe in the VikingFile first.

**Reply by lovelora123** [reply]
wait so it will be uploaded?

**Reply by lovelora123** [reply]
plz msg when it is cause I wanna play

**Reply by MonkeyDLuffy-MOD** [reply]
Update the link in VikingFiles, took upto an hour to show the changes.

**Reply by lovelora123** [reply]
so is the dlc there now will I be able to access it ?

**Reply by Admin-ToxicGames** [reply]
Yes

**Reply by lovelora123** [reply]
I downloaded previously from Akirabox and now I have my save file I just need to put ,my safe file in the game right 
?

**Comment by harivamsh234-web** [comment]
im not able to download it, the file is 53gb, everytime it comes to 30gb it gets cancelled suddenly, what should I do now, pls help

**Comment by RanaMahad** [comment]
I want to download with the dlc, how can I do that?

**Reply by Admin-ToxicGames** [reply]
Use Viking link

**Reply by RanaMahad** [reply]
VikingLink is broken, fails at 16GB, even with a download manager

**Reply by Admin-ToxicGames** [reply]
Just download it again it will fetch it again from there.

**Reply by RanaMahad** [reply]
thankyou it worked

**Comment by damndrums** [comment]
Overture please.
it's been updated on Appstore

**Comment by Star4999** [comment]
Hello admin, some people are writing it has malware is it true

**Comment by kshitijbansal09** [comment]
The game is not resuming from where it failed please help

**Reply by Vaibhav-rb** [reply]
lam facing a downloading issue

**Comment by Sharafooo** [comment]
is the dlc available

**Reply by Admin-ToxicGames** [reply]
Viking link
